.github/workflows/macos_ci_jobs.yml

```diff
@@ -135,23 +135,20 @@ jobs:
         ./tests/ci/run_accp_basic_tests.sh --lcov-ignore inconsistent
         ./tests/ci/run_accp_test_integration.sh
   # JDK 8 FIPS job.
```

Comment:
Missed this comment across all the FIPS variations.

Reply:
Thanks for catching it.